keps/sig-node/4381-dra-structured-parameters/README.md

```diff
@@ -557,278 +532,231 @@ reconstructed by the driver. This has no effect on already allocated claims
-  # "sharing" is a configuration parameter that does not
-  # get translated into the selector below.
-  sharing:
-    strategy: TimeSliced
-```
-
-Note that all fields in this CRD can be fully validated since it is owned by
-the DRA driver itself. This includes value ranges that are specific to the
-underlying hardware. There's no risk of using invalid attribute names because
-only the fields shown here are valid.
-
-With this CRD in place, a DRA driver controller is able to convert instances of
-it into a generic, in-tree `ResourceClaimParameters` object that the scheduler
-is able to understand.
-
-For the example above, the converted object would look as follows:
-
-```yaml
-kind: ResourceClaimParameters
-apiVersion: resource.k8s.io/v1alpha2
-
-metadata:
-  # This cannot be the same as my-parameters because parameter objects with a different
-  # type might also use it. Instead, the original object gets linked to below.
-  name: someArbitraryName
-  namespace: user-namespace
-
-generatedFrom:
-  name: my-parameters
-  kind: CardParameters
-  apiGroup: dra.example.com
-  uid: foobar-uid
-
-vendorParameters:
-  # A vendor can put any kind of object here to pass the configuration
-  # parameters down to the kubelet plugin. In this case, the vendor
-  # driver simply copied the entire CR. It could also be some
-  # separate, smaller configuration type.
-  #
-  # Beware that ResourceClaimParameters have separate RBAC rules than
-  # the vendor CRD, so information included here may get visible
-  # to more users than the original CRD. Both objects are in the same
-  # namespace.
-  kind: CardParameters
-  apiVersion: dra.example.com/v1alpha1
-  metadata:
-    name: my-parameters
-    namespace: user-namespace
-    uid: foobar-uid
-  ...
-  spec:
-    minimumRuntimeVersion: v12.0.0
-    minimumMemory: 32Gi
-    sharing:
-      strategy: TimeSliced
-
-driverRequests:
-- driverName: cards.dra.example.com
-  requests:
-  # Each entry here is a request for one resource.
-  - namedresources:
-      # Selectors are CEL expressions with access to the attributes of the named resource
-      # that is being checked for a match.
-      selector: |-
-        attributes.version["runtimeVersion"].isGreaterThan(semver("12.0.0")) &&
-        attributes.quantity["memory"].isGreaterThan(quantity("32Gi"))
+  # The node name indicates the node.
+  # Each driver on a node provides pools of devices for allocation,
+  # with unique device names inside each pool.
+  # Usually, but not necessarily, that pool name is the same as the
+  # node name.
+  nodeName: worker-1
+  poolName: worker-1
+  driverName: gpu.dra.example.com
+  sharedCapacity:
+  - name: gpu-0-memory-block-0
+    capacity: 1
+  - name: gpu-0-memory-block-1
+    capacity: 1
+  - name: gpu-0-memory-block-2
+    capacity: 1
+  - name: gpu-0-memory-block-3
+    capacity: 1
+  devices:
+  - name: gpu-0
+    attributes:
+    - name: memory
+      quantity: 40Gi
+    sharedCapacityConsumed:
```

Comment:
```suggestion
    consumesSharedCapacity:
```
?

Reply:
tried that, feedback from api review (well, Tim), was "nouns not verbs" for the fields. So, `sharedCapacityConsumed` is a noun ("shared capacity") with a adjective qualifier ("consumed"). There's probably some fancy grammar term for an adjective that is the past tense of a verb, but I don't know it.